integration/env/common.go

```diff
@@ -44,33 +54,41 @@ func init() {
 	if circleSHA == "" {
 		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarCircleSHA))
 	}
-	testedVersion = os.Getenv(EnvVarTestedVersion)
-	if testedVersion == "" {
-		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarTestedVersion))
-	}
-	token = os.Getenv(EnvVarGithubBotToken)
-	if token == "" {
-		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarGithubBotToken))
-	}
 
-	var err error
-	params := &framework.VBVParams{
-		Component: "chart-operator",
-		Provider:  "aws",
-		Token:     token,
-		VType:     TestedVersion(),
-	}
-	versionBundleVersion, err = framework.GetVersionBundleVersion(params)
-	if err != nil {
-		panic(err.Error())
+	testedCustomResource = os.Getenv(EnvVarTestedCustomResource)
+	if testedCustomResource == "" {
+		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarTestedCustomResource))
 	}
 
-	if VersionBundleVersion() == "" {
-		if strings.ToLower(TestedVersion()) == "wip" {
-			log.Println("WIP version bundle version not present, exiting.")
-			os.Exit(0)
+	if testedCustomResource == ChartConfigCustomResource {
```

Comment:
So instead of doing some hidden conditional routing in the code again, why don't we execute the test we want to execute? We could maybe have tests for the resources or functionality we want to test and isolate it from the rest which is not of interest. We define a test dir already, so this should maybe tell what to execute and the code would be cleaner.